### ISSUE TITLE
Missing maintainer scripts

### ISSUE THREAD

**Issue by ltabis** [kornelski/cargo-deb#69]
I add a bit of trouble lately creating debian packages using `cargo deb`: while I specified in my cargo manifest the `maintainer-scripts` field, none of my scripts where being picked-up by the program. In fact: the path I used was erroneous. Unfortunately, `cargo deb` did not provide any useful log/message about the matter.

Could it be interesting to display a warning when no files are found for maintainer scripts if the `maintainer-scripts` field is present in the cargo manifest ?

If you think it could be useful, I would be happy to make a pull request.

**Comment by kornelski** [comment]
Yes, it'd be useful. I think it could even fail the build if they're missing. But this may need to be checked after the build, so that build.rs has a chance to generate them.

**Comment by ltabis** [comment]
> But this may need to be checked after the build

Alright, I will take a look then !

**Comment by ximon18** [comment]
Related issue? https://github.com/kornelski/cargo-deb/issues/30

**Comment by ltabis** [comment]
> Related issue? #30

More or less, this issue is related to eventual missing maintainer scripts, #30 is for missing assets. Emitting a warning for missing assets also seems like a good idea to me.

**Comment by kornelski** [comment]
`RUST_LOG=debug cargo deb` will print more, including paths of scripts it wants.
